### ISSUE TITLE
Label changes not saved over VPN

### ISSUE THREAD

**Issue by mkbassett** [mbari-org/vars-gridview#75]
Over the last couple of weeks Megan and Kris have made changes in Gridview from home while on VPN. It appears that the label changes were not saved, but the "verifier" was updated to our respective names. The attached images show that both Megan and Kris are the verifiers of the highlighted images, but the concept was not changed to "Tomopterid eggcase". We both went in separately and made these changes on different days and neither stuck. 
<img width="1911" alt="Screenshot 2024-01-30 at 9 18 32 AM" src="https://github.com/mbari-org/vars-gridview/assets/39421936/8c2a5fd6-2d82-4bdb-b543-bdb1dcbf9ee4">
<img width="1912" alt="Screenshot 2024-01-30 at 9 18 20 AM" src="https://github.com/mbari-org/vars-gridview/assets/39421936/88226b2f-b805-4d5f-9815-bf9dcfa5869a">

**Comment by kevinsbarnard** [comment]
Ah, I think I know what's going on here -- I believe this is the same issue as #72, where there are two bounding boxes (one for the animal, one for the eggcase) tied to the same annotation, so changing the concept applies to both bounding boxes.

As a reference, this was why we originally went with the structure of having the concept be a field within the observation, and the part be a field (the `to_concept`) within the bounding box association. Under that view, there would be a single annotation (`concept = 'Tomopteris'`), where one bounding box is for the whole animal (`to_concept = 'self'`) and the other is for the eggcase (`to_concept = 'eggcase'`):
```
Tomopteris
    bounding box | self | {...}
    bounding box | eggcase | {...}
```

**Comment by kwalz** [comment]
Ah, that's it, Kevin. Thank you for looking into it so quickly for us. This gets me every time with the multiple boxes associated with one line of annotation in VARS.

**Comment by kevinsbarnard** [comment]
Sure thing! I think it would be good to discuss this at one of the VARS meetings, and even if we stick with the current structure, maybe I could add something to GridView to make this more apparent/less of a gotcha.

**Comment by kwalz** [comment]
Yes I think that is where the confusion is, the neighboring box in the same image doesn't get an updated concept name in the gridview thumbnail view--it still had Tomopteris so we are assuming only one box label changed to tomopterid eggcase, until we reload the image set. If the neighboring label changed at the same time in the thumbnail view, we would remember that this is one of those occasions where there are two (or more) box associations on the same line of annotation.

**Comment by lonnylundsten** [comment]
@kwalz @kevinsbarnard OK to close, right?

**Comment by kevinsbarnard** [comment]
@lonnylundsten Yes, this is part of a separate issue (that's linked in case we need to revisit this comment chain).
